proto/gds/peers/v1/peers.proto

```diff
@@ -32,7 +37,7 @@ service PeerManagement {
 // Used to filter the peers that are returned. If no filters are specified then all
 // known peers on the remote replica are returned.
 message PeersFilter {
-    repeated string region = 1;      // Specify the region(s) to return the peers for.
+    repeated string region = 1;      // Specify the region(s) to return the peers for. Only effects PeersList not PeersStatus
```

Comment:
Thank you so much for adding this comment, it's necessary to clarify what the filter does!